Scala/syntax_test_scala.scala

```diff
@@ -1189,3 +1189,38 @@ def <(a: Int) = 42
      &amp;
 //   ^^^^^ constant.character.entity.xml - meta.tag.xml
    </foo>
+
+// annotation examples from: http://www.scala-lang.org/files/archive/spec/2.11/11-annotations.html
+@deprecated("Use D", "1.0") class C { ... }
```

Comment:
This still needs to be addressed. The `meta` scope should probably be tested on a few of the different forms to make sure it terminates properly.